src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -552,11 +554,14 @@ void ComputeTimeDerivative<Metavariables>::
 
       // Store the boundary data on this side of the mortar
       db::mutate<all_mortar_data_tag>(
-          box, [&mortar_id, &temporal_id, &boundary_data_on_mortar ](
+          box, [&mortar_id, &temporal_id, &boundary_data_on_mortar,
+                &integration_order](
                    const gsl::not_null<typename all_mortar_data_tag::type*>
                        all_mortar_data) noexcept {
-            all_mortar_data->at(mortar_id).local_insert(
-                temporal_id, std::move(boundary_data_on_mortar));
+              auto& mortar_data = all_mortar_data->at(mortar_id);
+              mortar_data.integration_order(integration_order);
```

Comment:
Note to me: we do this operation in `ApplyBoundaryCorrections.hpp` (once the LTS support there is merged), which I think means we won't need the extra compatibility functions.